examples/05.submitOrder.js

```diff
@@ -30,10 +30,13 @@ const dvfConfig = {
   // Submit an order to sell 0.3 Eth for 200 USDT per 1 Eth
   const symbol = 'ETH:USDT'
   const amount = -0.3
-  const price = 200
   const validFor = '0'
   const feeRate = ''
 
+  // Gets the price from the order book api and cuts 5% to make sure the order will be settled
+  const orderBookPrice = await getPriceFromOrderBook();
```

Comment:
Since the next example depends on the settlement of this order and the price here was fixed, I'm just getting the current price from the API and subtracting 5% to make sure the order will be settled.